### ISSUE TITLE
💡 [Feature Request]Looting system rewrite + proper world drops

### ISSUE THREAD

**Issue by Zyres** [AscEmu/AscEmu#777]
**Description**: 

Ask Appled for more information

**Comment by Appled** [comment]
Updated the issue

**Comment by Zyres** [comment]
Any update on this? There was a lot of changes lately to loot?

**Comment by aaron02** [comment]
database misses loot groups and special loot flags for example hardmode loot or fishing junk loot

**Comment by Appled** [comment]
Yeah loot groups are still missing and that requires tons of db work. Atm iirc aaron hardcoded bosses to drop max 2 epics or so which works for now. Also, world drops are definitely not high prio so I'm dropping this issue to low priority.
